{{cookiecutter.project_slug}}/.github/workflows/build-test-publish.yml

```diff
@@ -52,12 +45,12 @@ jobs:
       - name: test
         run: make tests
       - if: github.event_name == 'push' && github.ref == 'refs/heads/master'
-        name: deploy
+        name: push
```

Comment:
minor: the canonical name for deploying is deploy not push.

Reply:
Right. I get confused with the action of deploying as in ``docker stack deploy`` ... IMO this is a ``push`` (or ``publish`` or ``release``) and what the deployer service does is actually a ``deploy``. 
I will bring that up as a suggestion to a change also in osparc-simcore and other repos.